tests/basilisp/test_core_fns.lpy

```diff
@@ -1096,6 +1096,10 @@
     {:a 1 :b 2} [:a :b] [1 2]
     {:a 0 :b 1} [:a :b] (range)))
 
+(deftest merge-with-test
```

Comment:
Can you add a test with just no arguments besides `f` and only a `nil` argument? Both should be `nil`.

Reply:
Done.